conference.js

```diff
@@ -1827,5 +1773,37 @@ export default {
         if(room) {
             room.sendApplicationLog(JSON.stringify({name, value}));
         }
+    },
+    /**
+     * Disconnect from the conference and optionally request user feedback.
+     * @param {boolean} [requestFeedback=false] if user feedback should be
+     * requested
+     */
+    hangup (requestFeedback = false) {
+        const errCallback = (err) => {
```

Comment:
Why not `function errCallback(err)`? The arrow function is still an anonymous function. And the whole construct of assigning an anonymous function to a local variable looks not as simple to me.